dazel.py

```diff
@@ -100,6 +101,7 @@ def __init__(self, instance_name, image_name, run_command, docker_command, docke
     @classmethod
     def from_config(cls):
         config = cls._config_from_file()
```

Comment:
Please pass the filename/env_var params here as well, and remove default values from the function. It will make things more explicit.

Reply:
👍